courses/machine_learning/deepdive2/image_classification/labs/2_mnist_models.ipynb

```diff
@@ -0,0 +1,691 @@
+{
```

Comment:
<p>Is there a reason calling test.py was deleted?</p><p>!python3 -m mnist_trainer.trainer.test</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I can't get it to work. Getting relative import fun. It may be a Python 2/3 issue...</p>